draft-selander-lake-authz.md

```diff
@@ -337,6 +337,21 @@ info = (
 )
 ~~~~~~~~~~
 
+## Stateless Operation of V
+
+V may act statelessly in respect to U.
+Once V receives the EDHOC message_1 from U, it forwards it unmodified to W in the form of a Voucher Request.
+V encapsulates the internal state that it needs to respond to U and sends it to W together with EDHOC message_1.
```

Comment:
Alternatively the clarification could be here:
> At this point, V can simply drop the EDHOC session that was initiated by U.

Reply:
@geonnave Which text should be replaced? Please clarify your proposal, for example using OLD .... NEW .... like this:

OLD

> V encapsulates the internal state that it needs to respond to U and sends it to W together with EDHOC message_1.

NEW

> At this point, V can simply drop the EDHOC session that was initiated by U.

Reply:
@gselander Malisa sent a fix that addressed these comments, I just forgot to "resolve" the conversation.